platform/setup.py

```diff
@@ -6,6 +6,7 @@
     'Django>=1.8,<1.9',
     'django-filter',
     'djangorestframework',
+    'drf-nested-routers',
```

Comment:
It would be nice form to have an issue to add this as a dep before this PR is merged.